crates/uv-resolver/src/lock/requirements_txt.rs

```diff
@@ -496,8 +496,6 @@ fn conflict_marker_reachability<'lock>(
                     queue.push(child_edge.target());
                 }
             }
-
-            queue.push(child_edge.target());
```

Comment:
I think this was just an oversight. This push is in the `match` above, with proper gating.